android/clientlib-ktx/src/main/java/com/solana/mobilewalletadapter/clientlib/MobileWalletAdapter.kt

```diff
@@ -15,6 +15,7 @@ import java.util.concurrent.TimeUnit
 import java.util.concurrent.TimeoutException
 
 class MobileWalletAdapter(
+    connectionIdentity: ConnectionIdentity? = null,
```

Comment:
should this be mandatory (non-optional)? what is the use case for not providing this during construction?

Reply:
hmm, using it the old way I think

Reply:
In the interest of backwards compatibility, should this be added as the last argument to the constructor?

Reply:
Actually this is a good idea